doc/rados/operations/pools.rst

```diff
@@ -604,47 +604,38 @@ You may set values for the following keys:
                  from these replies. This approach sacrifices resources in 
                  exchange for better performance. This flag is supported only 
                  for erasure-coded pools.  :Type: Boolean :Defaults: ``0``
-
 .. _scrub_min_interval:
 
 .. describe:: scrub_min_interval
-
-   The minimum interval in seconds for pool scrubbing when
-   load is low. If it is 0, the value osd_scrub_min_interval
-   from config is used.
+   
+   :Description: Sets the minimum interval (in seconds) for pool scrubbing when load is low. If the default value of ``0`` is in effect, then the value of ``osd_scrub_min_interval`` from central config is used.
```

Comment:
for successive scrubs of the pool's PGs when the load is low.

Reply:
Accepted.